packages/lib/modules/tokens/TokenRow/TokenRow.tsx

```diff
@@ -216,6 +219,9 @@ export default function TokenRow({
     variant: 'secondary',
   }
 
+  const { isAnyTokenWithoutPrice, tokenPriceTip } = usePoolTokenPriceWarnings()
```

Comment:
What is the difference between `isAnyTokenWithoutPrice` and `isTokenPriceMissing`?